build/config.props

```diff
@@ -6,7 +6,7 @@
     <!-- We need to update this netcoreassembly build number with every milestone to workaround any breaking api
     changes we might have made-->
     <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">2</NetCoreAssemblyBuildNumber>
```

Comment:
nit: should this go back to 0?

Reply:
I think it should...

Reply:
I don't think it matters too much, as long as it's a version that's different from the ones we have shipped. 

was it 0 at any point during the life of the 4.5 version?

Reply:
is this still used in the assembly version? I was assuming that it wasn't used at all. Now I'm more confused as to what it should be. Maybe it should go to 3 then?

Reply:
Yeah it is still used. 
We basically have a different one each preview to work around the issues we have with the sdk/cli insertions. 

Not sure if we have ever used a 0, but afaik it should be fine as long as it's different from what we've had before. 

/cc 
@rrelyea